streams/src/main/java/org/apache/kafka/streams/state/internals/MeteredWindowStore.java

```diff
@@ -116,7 +114,7 @@ public void put(final K key,
                     final long windowStartTimestamp) {
         final long startNs = time.nanoseconds();
         try {
-            inner.put(keyBytes(key), serdes.rawValue(value), windowStartTimestamp);
+            wrappedStore().put(keyBytes(key), serdes.rawValue(value), windowStartTimestamp);
```

Comment:
This is ugly -- we should have access to the wrapped store directly via inheritance. (also below)